model/src/main/kotlin/VcsType.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Copyright (C) 2019 Bosch Software Innovations GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonValue
+
+/**
+ * An class for Version Control System types. Each type has one or more [aliases] associated to it, where the first
+ * alias is the definite name.
+ */
+data class VcsType(val aliases: List<String>) {
```

Comment:
I this was an enum extensibility (without contributing) would be undermined a bit, but still shall we make it an enum as it seems quite a good fit. I think it is unlikely someone implement a new VCS which doesn't fit into ORT.

Reply:
That's exactly the discussion @mnonnenmacher and I had about my [enum proposal](https://github.com/heremaps/oss-review-toolkit/pull/1601). While we also agreed that an `enum class` is nice here and it's not very likely that someone needs to implement a VCS via a plugin that cannot be included into ORT core (e.g. because it's a commercial VCS), it still felt like an unnecessary limitation and hence I came up with this PR as a (better?) alternative.

Reply:
Oh - I wasn't aware of that discussion - will try to find it.

Reply:
I just realized most of that discussion happened in a direct Slack chat... sorry :sad:

Reply:
no problem, let's keep the non-enum approach and continue discussion based on that assumption.